.github/workflows/unit-test.yml

```diff
@@ -116,10 +110,8 @@ jobs:
               with:
                   custom-cache-suffix: $(/bin/date -u --date='last Mon' "+%F")
 
-            - name: Install npm dependencies
-              run: |
-                  npm ci
-                  npm run build
+            - name: Npm build
```

Comment:
Could we include this in the composite workflow instead of having to include it within each workflow?

We could add a new option for the composite workflow (maybe `run-build`?) with a default of `false` that would run `npm run build` when `true` is passed.

Reply:
I don't think this should be added in the composition action. Separating them could help us monitor the time difference between installing the dependencies and building the packages. Also, I think being a bit more explicit helps.